client/action.cpp

```diff
@@ -1,7 +1,7 @@
 /* BEGIN_COMMON_COPYRIGHT_HEADER
  * (c)LGPL2+
  *
- * LXDE-Qt - a lightweight, Qt based, desktop toolset
+ * LXQt - a lightweight, Qt based, desktop toolset
```

Comment:
The `http://razor-qt.org` is also outdated :)

Reply:
i know - second run :P